src/Composer/Satis/Builder/PackagesBuilder.php

```diff
@@ -83,6 +83,8 @@ private function dumpPackageIncludeJson(array $packages)
         rename($this->filenamePrefix, $filenameWithHash);
         $this->output->writeln("<info>wrote packages json $filenameWithHash</info>");
 
+        $this->clean($this->outputDir . '/include', "all\$$hash.json");
```

Comment:
the consequence here is to delete the file you've just created.

```
$this->filenamePrefix = $this->outputDir.'/include/all'; #at construction
$filenameWithHash = $this->filenamePrefix.'$'.$hash.'.json'; #at dump
```
Then, the clean() unlinks `$this->outputDir. '/include/' . "all\$$hash.json"`

This breaks the Packages Builder tests.